src/mixins/allowable-actions-mixin.ts

```diff
@@ -170,8 +178,7 @@ export default class AllowableActionsMixin extends Vue {
       case AllowableActions.VOLUNTARY_DISSOLUTION: {
         // NB: specific entities are targeted via LaunchDarkly
         const ff = !!GetFeatureFlag('supported-dissolution-entities')?.includes(this.getLegalType)
-        // FUTURE: check dissolution type
-        return (ff && this.isAllowedFiling(FilingTypes.DISSOLUTION))
+        return (ff && this.isAllowedFiling(FilingTypes.DISSOLUTION, FilingSubTypes.DISSOLUTION_VOLUNTARY))
```

Comment:
As above, I added the subtype check. This exposed a bug (reported in bcgov/entity#20619) where VD was incorrectly allowed when Admin Dissolution was allowed.